lib/BuildSystem/LaneBasedExecutionQueue.cpp

```diff
@@ -393,8 +423,27 @@ class LaneBasedExecutionQueue : public BuildExecutionQueue {
     posix_spawn_file_actions_init(&fileActions);
 
     // Open /dev/null as stdin.
-    posix_spawn_file_actions_addopen(
-        &fileActions, 0, "/dev/null", O_RDONLY, 0);
+//    posix_spawn_file_actions_addopen(
+//        &fileActions, 0, "/dev/null", O_RDONLY, 0);
```

Comment:
Probably want to hold off merge until we can fix this. :)

Reply:
Oops.